node/async-api-converter/README.md

```diff
@@ -0,0 +1,70 @@
+# @thing-description-playground/**TD_TO_ASYNCAPI**
```

Comment:
I think this line needs to be adapted, right?

Reply:
Done